File: Week1Pra/points.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    const int MINE = 2;
    int points = get_int("how many points did you lose? ");

    if (points > MINE)
    {
        printf("You lost more points than me\n");
    }
    else if (points < MINE)
    {
        printf("You lost less points than me\n");
    }
    else
    {
        printf("You lost the same amount of points as me\n");
    }
}

File: Practise/Week6Pra/points.py
from cs50 import get_int

# Ask user how many points they lost
points = get_int("How many points did you lose?" )

# If user achieved higher
if points < 2:
    print("You lost fewer points than me.")

# If user achived lower
elif points > 2:
    print("You lost more points than me.")

# If user achieved the same points
else:
    print("You lost the same number of points as me")